rag/cli.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env python3
+"""Simple CLI for the RAG system."""
+
+import argparse
+import logging
+import json
+import sys
+import time
+from pathlib import Path
+from typing import Dict, Any
+from colorama import init, Fore, Style
+
+# Initialize colorama
+init(autoreset=True)
+
+from core.base import Pipeline
+from core.enhanced_pipeline import EnhancedPipeline
+from parsers.csv_parser import CSVParser, CustomerSupportCSVParser
+from embedders.ollama_embedder import OllamaEmbedder
+from stores.chroma_store import ChromaStore
+from utils.progress import LlamaProgressTracker, create_enhanced_progress_bar
+
+
+def setup_logging(level: str = "INFO"):
+    """Setup logging configuration."""
+    logging.basicConfig(
+        level=getattr(logging, level.upper()),
+        format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    )
+
+
+def load_config(config_path: str) -> Dict[str, Any]:
```

Comment:
i'm working on a top-level config lib that you'll need to get the rag config from